tailwind.config.js

```diff
@@ -113,7 +113,11 @@ module.exports = {
 
       orange: {
         500: '#F3610F'
-      }
+      },
+
+      warning: '#F7931A',
```

Comment:
Let's stick to using names of colors (I've added orange for example), seems more tailwind-ish and will prevent some confusion I think

Reply:
i'm down with that. I chose the names because I received feedback from Terence that we should be using `#F7931A` for warning and `#FF0000` for error. We could change out all warning/error colors in the dex with these colors, but there is a rebrand imminent, so it becomes a question of whether it's worth spending the time, only for the work to be deprecated soonishly. 

@afterburn and all, do you think the `#F7931A` should be something like `orange-warning` or a `orange-xxx`? what would the `xxx` be? it should be less than the current `orange-500` since `#F7931A` is a lighter shade of orange compared to `F3610F` (our orange-500). same question for the red #FF0000.

Reply:
Good question, personally I feel like we should hold off from using new colors until we're ready to implement them throughout the product (to prevent these kinds of issues), so as a compromise I guess we could leave them as separate colors for now (warning/error).